feature-libs/customer-ticketing/root/public_api.ts

```diff
@@ -0,0 +1,2 @@
+export * from './customer-ticketing-root.module';
+export * from './feature-name';
```

Comment:
not related to this, but rename `feature-libs/customer-ticketing/root/services/index.ts` to facade (asm is one of the old features that didn't get a revamp)


basically instead of having `services` as the folder in root, make it `facade`